analyze_model.py

```diff
@@ -101,12 +102,33 @@ def generate_heatmap(data, row_labels, col_labels, filename, output_dir):
         plt.show()
 
     @staticmethod
-    def generate_heatmap_from_results(heatmap_name: str, results: dict, save_results_path: str, verbose=False):
+    def export_results_to_csv(results, row_labels, col_labels, filename, output_dir, verbose=True):
```

Comment:
Also, I think it makes more sense for this to be in `ResourceManager` because it has things like `save_results` already, knows the save directory etc.
The flow would be: the experiment calls `AnalyzeModel` for actual results, and the experiment should then call it's internal `ResourceManager` to save to CSV.
What do you think?

Reply:
I don't think it should be in resource manager, as this is never loaded (like heatmaps). hence the name "Export" and not "save"